media/css/firefox/firstrun/firstrun.less

```diff
@@ -0,0 +1,155 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+html {
+    background: @mozIDBlue;
+}
+
+#outer-wrapper {
+    .aurora-bars();
+}
+
+#intro {
+    color: #fff;
+    padding-bottom: (@baseLine * 3);
+    text-align: center;
+}
+
+header {
+    h1 {
+        margin-bottom: 0;
+        .font-size(4rem);
```

Comment:
This seems to be `5rem` (from `.font-size(80px);`) in the variation in production.